libs/input-mapping/src/Table/Options/InputTableOptions.php

```diff
@@ -1,26 +1,38 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\InputMapping\Table\Options;
 
+use Keboola\InputMapping\Configuration\Table;
 use Keboola\InputMapping\Exception\InvalidInputException;
 use Keboola\InputMapping\Exception\TableNotFoundException;
 use Keboola\InputMapping\State\InputTableStateList;
 
+/** @phpstan-type ColumnType = array{
+ *     source: string,
+ *     type?: string,
+ *     destination?: string,
+ *     length?: integer,
+ *     nullable?: bool,
+ *     convertEmptyValuesToNull?: bool,
+ * }
+ * @phpstan-type ColumnDefinition = array{
+ *     source: string,
+ * }
+*/
```

Comment:
Kde jsou ty typy pouzite? :thinking:

Reply:
columntype je tady https://github.com/keboola/platform-libraries/blob/c330c9865dfd410d25c39c0761e92401b22d3e71/libs/input-mapping/src/Table/Options/InputTableOptions.php#L189 
columndefinition dávám pryč, to se neujalo